src/stemmabench/stemma_generator.py

```diff
@@ -90,9 +94,29 @@ def __repr__(self) -> str:
     def _apply_level(self, manuscript: str) -> List[str]:
         """Apply transformation on a single generation"""
         return [Text(manuscript).transform(self.config.variants,
-                                           meta_config=self.config.meta) 
+                                           meta_config=self.config.meta)
                                            for _ in range(self.width)]
 
+    def missing_manuscripts(self) -> Tuple[Dict[str, str], List[Tuple[str]]]:
+        """Remove some manuscripts from the tradition.
+        """
+        # Initialize a RNG.
+        rng = np.random.default_rng(self.random_state)
```

Comment:
why is this required ? Won't this affect the randomness of the process ?

Reply:
No, as long as it's set to "None," the "random_state" does not influence the randomness of the process.

Reply:
- [ ] Replace rng with choice